.gitignore

```diff
@@ -20,3 +20,6 @@ perun
 
 # Local perun configuration:
 .perun
+preprocessed.yml
```

Comment:
Out of curiosity: what is it?

Reply:
This is a temporary file which is being created when `FixFunctions` is called from `intrinsicsolver` package.